### DISCUSSION TITLE
G142D

### DISCUSSION THREAD

**Discussion by AngeliqueK** [hodcroftlab/covariants#217] [Q&A]
G142D is listed as a point mutation of B.1.617.2 on gvn.org
Is this incorrect?

**Comment by emmahodcroft** [comment]
Hi @AngeliqueK ! The position at Spike 142 seems to be problematic unfortunately - it changes back and forth many times within the Delta clade, and this seems to be some kind of sequencing issue, rather than real diversity - you can see how often it changes below:
https://nextstrain.org/groups/neherlab/ncov/21A.Delta?c=gt-S_142
![image](https://user-images.githubusercontent.com/14290674/132214686-17b53865-cf2d-4e27-b0d2-24a54d2129a8.png)

We've actually started masking this site in the official Nextstrain builds, and I will soon be masking it for the CoVariants builds as well. 

I hope that helps!

**Comment by AngeliqueK** [comment]
Thank you for your response! What’s interesting is the occurence of G142D in american mink sequences see https://pubmed.ncbi.nlm.nih.gov/34109885/

I wonder what other animals this mutation might help the virus jump to? 

Thanks for your help.

Angelique K.

**Reply by emmahodcroft** [reply]
Thanks Angelique! This is certainly interesting to read, and important to know. The S:142 mutation in Delta seems to be caused by a bad primer, but it's key to remember that this may be real in other circumstances. Thanks for letting us know about this, and we can consider how might be best to tackle this in the future.
